cli-commands/compiler/compiler.js

```diff
@@ -0,0 +1,46 @@
+const logsStore = require('./../../logs-store/logs-store');
+const utils = require('./../util');
+
+let compiler = require("./etherlime-workflow-compile/index");
+let Resolver = require("./etherlime-resolver/index");
+
+const run = async (defaultPath) => {
+    const initialRecordsCount = logsStore.getHistory().length;
+
+    try {
+        let resolverOptions = {
```

Comment:
Move the compilation with options into another function called `performCompilation`